### PR TITLE
Release notes for week of October 26

### PR DESCRIPTION
**Pull request by srpearce**
@mhoeber Please review. I need to add the info about partner doc for the certificates (first item under LMS).

### CODE REVIEW

**Comment by mhoeber** [comment]
@srpearce  looks good.  change logs for course team docs have new entry on certificate data in instructor dashboard

**Comment by srpearce** [comment]
@mhoeber Thanks. I included the info about the new certificate data in the Documentation section - did you see it? (doc_1027_2015.rst, line 9)

**Comment by mhoeber** [comment]
@srpearce  I see the course number override doc reference.  Don't see "Added the Reporting Certificate Data topic." (added yesterday to B&R guides)

**Comment by srpearce** [comment]
Oh, yep - am in the process of adding that.

**Comment by srpearce** [comment]
@mjfrey - FYI. You don't need to sign off on these. As soon as the release goes live, I'll send you the URL with the latest release notes.

@explorerleslie @lou-wang Can one of you review these, please?

File: en_us/release_notes/source/2015/discussions/disc_1027_2015.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+When a learner on a team created a post for members of that team only, a "This
+post is visible to everyone" message appeared at the end of the inline view of
+the post. This problem has been fixed. (TNL-3575)
```
Comment (explorerleslie) [inline en_us/release_notes/source/2015/discussions/disc_1027_2015.rst]: We shouldn't mention teams bug fixes -- the teams doc isn't even live yet, only two courses are using it.

File: en_us/release_notes/source/2015/lms/lms_1027_2015.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+* After you generate certificates for your course, you can find information
+  about the certificates on the Instructor Dashboard. You can see the number
+  and type of certificates that learners earned, or download a .csv file of
+  all certificates. For more information, see
+  :ref:`partnercoursestaff:Reporting Certificate Data` in *Building and
+  Running an edX Course* or :ref:`opencoursestaff:Reporting Certificate Data`
+  in *Building and Running an Open edX Course*.
```
Comment (explorerleslie) [inline en_us/release_notes/source/2015/lms/lms_1027_2015.rst:8]: @lou-wang please review.

File: en_us/release_notes/source/2015/studio/studio_1027_2015.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+Course teams can now add unlimited signatories to certificates. (SOL-1312)
```
Comment (explorerleslie) [inline en_us/release_notes/source/2015/studio/studio_1027_2015.rst]: @lou-wang is this true? How the heck are they all going to fix on the cert? It looks like from the ticket that PMs will need to know to keep it to 4, so maybe we don't want to publicize this removal?

File: en_us/release_notes/source/2015/discussions/disc_1027_2015.rst (hunk 1 above)
Comment (srpearce) [inline en_us/release_notes/source/2015/discussions/disc_1027_2015.rst]: OK, deleted.

**Comment by explorerleslie** [comment]
@srpearce :+1: , pending removal of teams bug fix and questions to @lou-wang about certs.

File: en_us/release_notes/source/2015/studio/studio_1027_2015.rst (hunk 3 above)
Comment (lou-wang) [inline en_us/release_notes/source/2015/studio/studio_1027_2015.rst]: Yeah, they took out the limit, but I would not advertise it per Leslie's comment. I also think we should still implement a slightly looser but still reasonable limit of 6, but that's a separate matter.

**Comment by lou-wang** [comment]
Also see my comment about pulling the signatory limit comment - otherwise :+1:

File: en_us/release_notes/source/2015/studio/studio_1027_2015.rst (hunk 3 above)
Comment (srpearce) [inline en_us/release_notes/source/2015/studio/studio_1027_2015.rst]: OK, deleted.

**Comment by srpearce** [comment]
Made changes suggested by Alison, Leslie, and Lou.

File: en_us/release_notes/source/2015/studio/studio_1027_2015.rst (hunk 3 above)
Comment (mhoeber) [inline en_us/release_notes/source/2015/studio/studio_1027_2015.rst]: Technically the limit is 100 because some int was needed.  @mattdrayer ^^

**Pull request merged**